src/components/ProcessedReleaseChangeDescription.tsx

```diff
@@ -90,7 +90,15 @@ const ProcessedReleaseChangeDescription = ({
         <>
           <Tag size="md" mb={2} rounded="full" colorScheme="primary">
             <Box as={GoTag} h={4} w={4} mr={1} />
-            <TagLabel>{getReleaseVersion(processedReleaseChange)}</TagLabel>
+            <TagLabel>
```

Comment:
Cool! I like this! 😄 

I have 2 suggestions:

- make the entire tag area a link/clickable.
- Add `:focus` and `:hover` styles

Only the link text is clickable, normally in an interface you can click on the whole button. Can you make it so that the whole tag area is clickable?

I'd also like `:hover` and `:focus` states for the tag area, so that when you hover over the badge, there's a slight color change. Would be nice to have a `focus-ring` on the outline of the badge when it's selected via keyboard.

Reply:
Oh, and the background of the tag could maybe change as well once you actually click on it with `:active`. 😄 

Chakra UI has the wanted props! 😄 

- `_hover`
- `_focus`
- `_active`

Source: https://chakra-ui.com/docs/features/style-props#pseudo

Reply:
Making the entire tag clickable seems much better indeed. I'll play around with it and push something during the week.

Reply:
> Making the entire tag clickable seems much better indeed. I'll play around with it and push something during the week.

Glad you agree. I look forward to the improvements. 😄

Reply:
Updated! Let me know what you think.